bin/shpec

```diff
@@ -1,41 +1,51 @@
-#!/usr/bin/env bash
-VERSION=0.1.2
-examples=0
-failures=0
-test_indent=0
-red="\033[0;31m"
-green="\033[0;32m"
-norm="\033[0m"
+#
```

Comment:
On 04/20/15 17:20, Ryland Herrick wrote:
> In bin/shpec <https://github.com/rylnd/shpec/pull/58#discussion_r28697899>:
>
>> @@ -1,41 +1,51 @@
>> -#!/usr/bin/env bash
>> -VERSION=0.1.2
>> -examples=0
>> -failures=0
>> -test_indent=0
>> -red="\033[0;31m"
>> -green="\033[0;32m"
>> -norm="\033[0m"
>> +#
>
> Please delete this line.

I see now why I kept that line in there.  It's entirely non-functional,
but it's related to vim syntax highlighting.

With the  `#! /usr/bin/env bash`, vim automatically does rich syntax highlights.
Those disappear when I remove the entire line.

When I keep just the plain `#` as the first line, we get some very basic highlights.

There is a way to fix it *and* get rid of that line:

Rename the script to `bin/shpec.sh` and create a packager and/or installer that removes the suffix.

The `.sh` suffix gives me full syntax highlighting, even without that line.